hardhat.config.js

```diff
@@ -0,0 +1,22 @@
+require("@nomiclabs/hardhat-etherscan");
+require('dotenv').config();
+
+module.exports = {
+  solidity: {
+    version: "0.6.12",
+    settings: {
+    optimizer: {
+        "enabled": false,
```

Comment:
```suggestion
        "enabled": true,
```
should we say here that we are optimized seeing as we deployed optimized contracts?

Reply:
@rheeger, as the contracts successfully verified with `"enabled": false`, I suspect that the contract artifacts that were deployed were compiled with the optimizer disabled. Do you happen to have the artifacts from deploy on your machine? If so, you can check the artifact json and towards the bottom there's a place where the compiler settings live:

![image](https://user-images.githubusercontent.com/5423313/101403716-3da14e80-389b-11eb-8eac-d2973daf323c.png)

Reply:
fair enough...i thought i had but maybe not. don't have the artifacts handy. This works for now.